Readme.md

```diff
@@ -0,0 +1,60 @@
+**Pre-requisite:**
+
+1. python2
+2. gcloud
+
+Follow these set-up before running the benchmarking script.
+
+1. Keep your envoy-binary in an accessible location, `$ENVOY_BINARY`.
+2. Keep all the scripts, Makefiles in a directory, `$SCRIPTS`. The scripts, Makefiles refer to the same files that are included in this benchmarking package.
```

Comment:
It sounds like SCRIPTS is set to the directory that is the root of the local clone of this repo. My inclination would be to change the default, and have benchmark.py infer it from the path to itself. See these pages:

https://stackoverflow.com/questions/3718657/how-to-properly-determine-current-script-directory-in-python

http://www.faqs.org/docs/diveintopython/regression_path.html